src/sentry/web/frontend/debug/mail.py

```diff
@@ -456,7 +464,29 @@ def build_usage_summary():
             int(random.weibullvariate(5, 1) * random.paretovariate(0.2)),
         )
 
-    def build_report():
+    def build_calendar_data(project):
```

Comment:
Alright, rebased and updated this — I didn't check the coverage before, but I'm pretty sure it looks a lot better now.